src/Interactive/Host/Interactive/Core/InteractiveHost.Service.cs

```diff
@@ -160,13 +160,13 @@ public void Dispose()
                 _serviceState = null;
             }
 
-            public Task<InteractiveHostPlatformInfo.Data> InitializeAsync(string replServiceProviderTypeName, string cultureName)
+            public Task<InteractiveHostPlatformInfo.Data> InitializeAsync(string replServiceProviderTypeName)
             {
                 // TODO (tomat): we should share the copied files with the host
                 var metadataFileProvider = new MetadataShadowCopyProvider(
                     Path.Combine(Path.GetTempPath(), "InteractiveHostShadow"),
                     noShadowCopyDirectories: s_systemNoShadowCopyDirectories,
-                    documentationCommentsCulture: new CultureInfo(cultureName));
+                    documentationCommentsCulture: CultureInfo.CurrentCulture);
```

Comment:
Is this actual the language that the doc comments should be displayed? If so, this should be CurrentUICulture which reflects the culture used for localization.

Reply:
Both cultures are the same. See https://github.com/dotnet/roslyn/pull/67677/files#diff-f3cb48bba334e574776b2f2ef9ee18b8825f9f6ce5943c30823090febd14afb7R32-R35

I'm not sure what the difference between UI culture and culture would be in the context of Interactive Window.

Reply:
CurrentCulture is for outputing and parsing user data (dates, numbers, etc), CurrentUICulture is for localization. I for example, have CurrentUICulture set to _en-AU__, but CurrentCulture set to _en-US_ so that I see US dates instead of Australian dates. A Japanese person living in the USA might have CurrentUICulture set to ja-JP, but CurrentCulture set to _en-US_.